Sync CHANGELOG with Calyptia-Fluentd v1.4.2 release (#1)
calyptia/calyptia-fluentd-notifications: CHANGELOG.md

# Calyptia Fluentd 1 changelog

## Release v1.4.2 - 2022/11/29

### News

> * Upgrade Rubies to 3.1.3/3.0.5

Reviewer: Should this be `Ruby`?

Author: Not sure but I sometimes see the plural form of Ruby like as this line.

Reviewer: Yeah it looks wrong - there is only one version on each target so I'd just use the singular.

# Calyptia Fluentd 1 changelog

## Release v1.4.0 - 2022/06/30

### News

* Upgrade Fluentd to v1.15.0

>   * In this version of Fluentd, it starts to handle YAML configuration.

Reviewer: ```suggestion
  * In this version of Fluentd, experimental support for handling YAML configuration is provided.
```

Author: We Fluentd maintainer also provides the detail document to use YAML configuration format: 
https://docs.fluentd.org/configuration/config-file-yaml

In Fluentd side, it's not experimental feature.

Reviewer: Fair enough, maybe:
```suggestion
* This version of Fluentd provides support for YAML configuration as well.
```

Author: Hmm, it's reasonable for me. I'll handle to modify with your suggestion later.

> * Use vanilla fluent-plugin-splunk-hec v1.12.13 which is resloved Ruby 3 compatibility issues

Reviewer: ```suggestion
* Use the vanilla fluent-plugin-splunk-hec v1.12.13 which resolves Ruby 3 compatibility issues
```

* elasticsearch v7.17.1 (update)
* fluent-plugin-elasticsearch v5.2.1 (update)
* ruby-kafka v1.4.0
* fluent-plugin-kafka v0.17.5 (update)
* prometheus-client v2.1.0
* fluent-plugin-prometheus v2.0.2
* aws-sdk-core v3.130.0
* aws-sdk-s3 v1.113.0
* fluent-plugin-s3 v1.6.1
* fluent-diagtool v1.0.1
* td-client v1.0.8
* fluent-plugin-td v1.1.0
* webhdfs v0.10.2
* fluent-plugin-webhdfs v1.5.0
* fluent-plugin-record-modifier v2.1.0
* fluent-plugin-rewrite-tag-filter v2.4.0
* fluent-plugin-sd-dns v0.1.0
* fluent-plugin-flowcounter-simple v0.1.0
* fluent-plugin-detect-exceptions v0.0.14
* fluent-plugin-metrics-cmetrics v0.1.1
* fluent-plugin-calyptia-monitoring v0.1.3
* fluent-plugin-splunk-hec v0
* fluent-plugin-cmetrics v0.1.0
* fluent-plugin-in-http-splunk-hec v0.1.0.rc2
* opensearch-ruby v1.0.0
* fluent-plugin-opensearch v1.0.2
* rdkafka v0.11.1

On Linux

* systemd-journal v1.4.2
* fluent-plugin-systemd v1.0.5
* fluent-plugin-utmpx v0.5.0

On Windows

* winevt_c v0.9.3
* fluent-plugin-parser-winevt_xml v0.2.4
* fluent-plugin-windows-eventlog v0.8.1

## Release v1.3.5 - 2022/02/28

### News

* Update Fluentd version to v1.14.5
* Including fixes for fluent-plugin-opensearch

> * Update Elasticsearch 8 compatible flunet-plugin-elasticsearch

Reviewer: ```suggestion
* Update to Elasticsearch 8 compatible fluent-plugin-elasticsearch
```

* async-http v0.56.5
* serverengine v2.2.4

### Bundled plugins and gems

* elasticsearch v7.15.0
* fluent-plugin-elasticsearch v5.1.4 (update)
* ruby-kafka v1.4.0
* fluent-plugin-kafka v0.17.3 (update)
* prometheus-client v2.1.0
* fluent-plugin-prometheus v2.0.2
* aws-sdk-core v3.123.0 (update)
* aws-sdk-s3 v1.107.0 (update)
* fluent-plugin-s3 v1.6.1
* fluent-diagtool v1.0.1
* td-client v1.0.8
* fluent-plugin-td v1.1.0
* webhdfs v0.10.2
* fluent-plugin-webhdfs v1.5.0
* fluent-plugin-record-modifier v2.1.0
* fluent-plugin-rewrite-tag-filter v2.4.0
* fluent-plugin-sd-dns v0.1.0
* fluent-plugin-flowcounter-simple v0.1.0
* fluent-plugin-detect-exceptions v0.0.14
* fluent-plugin-metrics-cmetrics v0.1.1
* fluent-plugin-calyptia-monitoring v0.1.3
* fluent-plugin-splunk-hec v0
* fluent-plugin-cmetrics v0.1.0.rc8
* rdkafka v0.11.1 (update)

On Linux

* systemd-journal v1.4.2
* fluent-plugin-systemd v1.0.5
* fluent-plugin-utmpx v0.5.0

On Windows

* winevt_c v0.9.3
* fluent-plugin-parser-winevt_xml v0.2.3
* fluent-plugin-windows-eventlog v0.8.1

## Release v1.3.1 - 2021/11/01

### News

* Update using Fluentd to v1.14.2 vanilla
* Update Elasticsearch related gems and plugin

>   * Note that this version blocks for communicating with OpenSearch. We'll support OpenSearch later with another plugin.

Reviewer: ```suggestion
  * Note that this version does not support OpenSearch.
```
